ports/mimxrt/modules/_boot.py

```diff
@@ -3,13 +3,31 @@
 # Note: the flash requires the programming size to be aligned to 256 bytes.
 
 import os
+import sys
 import mimxrt
+from machine import SDCard
 
 bdev = mimxrt.Flash()
-
 try:
     vfs = os.VfsLfs2(bdev, progsize=256)
 except:
     os.VfsLfs2.mkfs(bdev, progsize=256)
     vfs = os.VfsLfs2(bdev, progsize=256)
-os.mount(vfs, "/")
+finally:
```

Comment:
why does this need to go in a `finally`?

Reply:
fixed